packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -223,7 +223,7 @@ export default {
                         if (!verification.issuerName && offchainVerification.issuerName) {
                             verification.issuerName = offchainVerification.issuerName
                         }
-                        if (typeof verification.issuerVerified !== 'boolean' && typeof offchainVerification.issuerVerified === 'boolean') {
+                        if (!verification.issuerVerified && typeof offchainVerification.issuerVerified === 'boolean') {
```

Comment:
Here is the rule that we have in the backend in `legacy_onchain_verifier.go`:
```
if offChainIssuer.Verified && offChainIssuer.VerifiedBy != "" {
		// This is for HIN and SWITCH
		mergedIssuer.Name = offChainIssuer.Name
		mergedIssuer.Verified = true
		mergedIssuer.VerifiedBy = offChainIssuer.VerifiedBy
	}
```

Reply:
The difference to our implementation is, that the offchain result can set it to false.

Reply:
Should we adapt our logic to the backend logic?

Reply:
Yes, because the current logic, which is a (hum, hum...) a hack, goes only in one direction.  We do not plan for the db to contradict the contract in general.

Reply:
Adjusted acc.